Dockerfile-ubi

```diff
@@ -0,0 +1,38 @@
+# Build the manager binary
```

Comment:
Why can't we continue using - https://github.com/SumoLogic/sumologic-kubernetes-collection-helm-operator/blob/main/Dockerfile

Are there any changes needed to the Makefile, to incorporate this change?

Reply:
The Dockerfile can be used but we need it to be based on ubi base image. 

If we do that after release we will have an UBI based image. If that is okay with us surely we can make these changes on the Dockerfile.

Reply:
If we're not using non-ubi images anywhere then maybe this change could be made to the existing Dockerfile.